app.py

```diff
@@ -12,13 +13,10 @@
 
 # Load configuration
 config = get_config()
-
-# Setup logging based on configuration
-logging.basicConfig(
-    level=getattr(logging, config.logging.level),
-    format=config.logging.format
+coloredlogs.install(
+    level='INFO',
+    fmt=config.logging.format,
```

Comment:
```suggestion
coloredlogs.install(
    level=config.logging.level,
    fmt=config.logging.format,
)
```


logging is already set to INFO in config.json. Other than that, looks good. Thanks for the PR

Reply:
LGTM